packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-field-input/question.js

```diff
@@ -97,7 +97,34 @@ class MultiFieldInputQuestion extends Question {
 		return responseToSave;
 	}
 
-	// TODO - formatAnswerForSummary, getAction, consider Section isComplete...
+	/**
+	 * returns the formatted answers values to be used to build task list elements
+	 * @param {Journey} journey
+	 * @param {String} sectionSegment
+	 * @returns {Array.<Object>}
+	 */
+	formatAnswerForSummary(sectionSegment, journey) {
+		let summaryDetails = '';
+
+		this.inputFields.forEach((field, index) => {
+			const answer = journey.response.answers[field.fieldName];
+			if (answer !== undefined && answer !== null && answer !== '') {
+				summaryDetails += answer;
+
+				if (index < this.inputFields.length - 1) {
+					summaryDetails += ' ';
+				}
+			}
+		});
```

Comment:
Does this work the same as
```
const summaryDetails = this.inputFields.reduce((acc, cur) => {
  const answer = journey.response.answers[field.fieldName];
  if(!answer) return acc
  return [...acc, answer]
}, []).join(' ')
```
?

Reply:
Yeah the nested ternary took me a sec to parse but I don't think it's _un_reasonable